infra/core/search/search-private-link.bicep

```diff
@@ -0,0 +1,24 @@
+param name string
```

Comment:
I'd add descriptions to every property.  You can use chatgpt to generate them for you.

Reply:
Added!